compiler/src/main/kotlin/io/github/apl_cornell/viaduct/errors/InsecureDataFlowError.kt

```diff
@@ -21,23 +26,31 @@ class InsecureDataFlowError(
 
     override val description: Document
         get() {
-            if (!nodeLabel.confidentiality().flowsTo(to.confidentiality())) {
+            if (!context.lessThanOrEqualTo(
+                    nodeLabel.confidentialityComponent,
```

Comment:
You have flipped the arguments. It's better to use functions at `Label` level instead of component level for exactly this reason.

Reply:
Added A label level actsfor function that takes congruence so that this can be implemented